storage/namespace.go

```diff
@@ -277,6 +346,16 @@ func (n *dbNamespace) Bootstrap(
 	workers := pool.NewWorkerPool(int(math.Ceil(float64(runtime.NumCPU()) / 2)))
```

Comment:
Bootstraps are such low frequency and this will only create a chan of like 16 structs on a 32 CPU machine anyhow.  I doubt this allocation will hurt overall performance of the process in any way.  There are several magnitude larger allocations that occur during a bootstrap run.